tests/test_diffuse.py

```diff
@@ -0,0 +1,149 @@
+import imkar as ik
+import numpy as np
+import pyfar as pf
+import pytest
+import numpy.testing as npt
+
+
```

Comment:
add test for multidimensional inputs
np.vstack(([T60_1, T60_2, T60_3, T60_4], [T60_1, T60_2, T60_3, T60_4]))